### ISSUE TITLE
Possible bug with HEMCO at 0.25 degree resolution

### ISSUE THREAD

**Issue by drewpendergrass** [geoschem/HEMCO#213]
### Name and Institution (Required)

Name: Drew Pendergrass
Institution: Harvard U

### Confirm you have reviewed the following documentation

- [X] [Support guidelines](https://hemco.readthedocs.io/en/stable/reference/SUPPORT.html)
- [X] [User manuals](https://hemco.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/related-docs.html)
- [X] [Debugging GEOS-Chem and HEMCO errors](https://hemco.readthedocs.io/en/stable/geos-chem-shared-docs/supplemental-guides/debug-guide.html)
- [X] [Current and past Github issues](https://github.com/geoschem/HEMCO/issues)

### Description of your issue or question

Please provide as much detail as possible. Always include the HEMCO version number and any relevant configuration and log files.

For both the latest HEMCO (3.6.2) and HEMCO 3.0.0, I've had issues with running HEMCO at quarter degree both with the HEMCO Config file that ships as an example (and for integration tests, I assume) and with a personal HEMCO Config I've used in GEOS-Chem. The model seems to crash as soon as it tries to read the grid latitudes specification (HEMCO_sa_Grid.rc). 

This is the error from 3.6.2 with the shipped HEMCO Config file:
```console
Reading entire HEMCO configuration file: HEMCO_sa_Config.rc
  
Reading entire HEMCO configuration file: HEMCO_Config.rc
 Option is already defined - use original value of /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/HEMCO and ignore the following value: ROOT:                         {DATA_ROOT}/HEMCO
 Option is already defined - use original value of OutputDir/HEMCO_sa_diagnostics and ignore the following value: DiagnPrefix:                  ./OutputDir/HEMCO_diagnostics
 Option is already defined - use original value of 00000100 000000 and ignore the following value: DiagnFreq:                    End
  
Reading entire HEMCO configuration file: /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/HEMCO/Configs/HEMCO_Config.rc.gmao_metfields
At line 1360 of file /n/holylfs05/LABS/jacob_lab/dpendergrass/emissions/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90
Fortran runtime error: End of record

Error termination. Backtrace:
#0  0x2ab07ad0cbd7 in write_character
	at ../.././libgfortran/io/write.c:1403
#1  0x2ab07ad11546 in list_formatted_write_scalar
	at ../.././libgfortran/io/write.c:1876
#2  0x2ab07ad04a05 in wrap_scalar_transfer
	at ../.././libgfortran/io/transfer.c:2369
#3  0x2ab07ad04a05 in wrap_scalar_transfer
	at ../.././libgfortran/io/transfer.c:2346
#4  0x407425 in set_grid
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/emissions/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90:1341
#5  0x4157e1 in __hcoi_standalone_mod_MOD_hcoi_sa_init
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/emissions/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90:325
#6  0x4165bc in __hcoi_standalone_mod_MOD_hcoi_standalone_run
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/emissions/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90:194
#7  0x4031ab in hemco_standalone
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/emissions/CodeDir/src/Interfaces/Standalone/hemco_standalone.F90:136
#8  0x402e06 in main
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/emissions/CodeDir/src/Interfaces/Standalone/hemco_standalone.F90:21

real	0m0.216s
user	0m0.094s
sys	0m0.040s

This is the issue with 3.0.0

Reading entire HEMCO configuration file: /n/holyscratch01/external_repos/GEOS-CHEM/gcgrid/data/ExtData/HEMCO/Configs/HEMCO_Config.rc.gmao_metfields
At line 1351 of file /n/holylfs05/LABS/jacob_lab/dpendergrass/hemco3/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90
Fortran runtime error: End of record

Error termination. Backtrace:
#0  0x2ad253644bd7 in write_character
	at ../.././libgfortran/io/write.c:1403
#1  0x2ad253649546 in list_formatted_write_scalar
	at ../.././libgfortran/io/write.c:1876
#2  0x2ad25363ca05 in wrap_scalar_transfer
	at ../.././libgfortran/io/transfer.c:2369
#3  0x2ad25363ca05 in wrap_scalar_transfer
	at ../.././libgfortran/io/transfer.c:2346
#4  0x407219 in set_grid
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/hemco3/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90:1332
#5  0x413d8e in __hcoi_standalone_mod_MOD_hcoi_sa_init
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/hemco3/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90:325
#6  0x4149bc in __hcoi_standalone_mod_MOD_hcoi_standalone_run
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/hemco3/CodeDir/src/Interfaces/Standalone/hcoi_standalone_mod.F90:194
#7  0x40303b in hemco_standalone
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/hemco3/CodeDir/src/Interfaces/Standalone/hemco_standalone.F90:136
#8  0x402c96 in main
	at /n/holylfs05/LABS/jacob_lab/dpendergrass/hemco3/CodeDir/src/Interfaces/Standalone/hemco_standalone.F90:21
```
When I ran with a 0.5 degree grid I was able to resolve the issues.

**Comment by msulprizio** [comment]
Hi @drewpendergrass. I also came across this issue. It should be fixed by https://github.com/geoschem/HEMCO/pull/207 which will be included in HEMCO 3.7.0.

**Comment by drewpendergrass** [comment]
Excellent, thank you so much!

**Comment by yantosca** [comment]
Closed by #207

**Comment by yantosca** [comment]
We can close this issue as PR #207 has been merged into the HEMCO 3.7.0 development stream.
